<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchClass</class>
 <widget class="QDialog" name="SearchClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search for Contacts or Groups</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabUsers">
      <attribute name="title">
       <string>Users</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <property name="margin">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="QStackedWidget" name="stackedWidgetUsers">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="pageUsersSearchCondition">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBoxUsersSearchMode">
             <property name="title">
              <string>Search Mode</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <spacer name="horizontalSpacer_7">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QRadioButton" name="radioButtonUsersMatchExactly">
                  <property name="text">
                   <string>&amp;Exactly Match</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_8">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QRadioButton" name="radioButtonUsersMatchWildcard">
                  <property name="text">
                   <string>&amp;Wildcard Match</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_9">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="groupBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string/>
             </property>
             <layout class="QGridLayout" name="gridLayout_15">
              <item row="1" column="0">
               <widget class="QStackedWidget" name="stackedWidgetUsersSearchCondition">
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <widget class="QWidget" name="pageUsersSearchMatchExactly">
                 <layout class="QGridLayout" name="gridLayout_5">
                  <item row="0" column="0">
                   <widget class="QLabel" name="labelUserID">
                    <property name="text">
                     <string>User ID:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLineEdit" name="lineEditUserID"/>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="labelNickname">
                    <property name="text">
                     <string>Nick Name:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLineEdit" name="lineEditNickname"/>
                  </item>
                  <item row="4" column="0">
                   <spacer name="verticalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>97</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="pageUsersSearchMatchWildcard">
                 <layout class="QGridLayout" name="gridLayout_4">
                  <item row="0" column="0">
                   <widget class="QLabel" name="labelGender">
                    <property name="text">
                     <string>Gender:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>Age:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QComboBox" name="comboBoxGender">
                    <item>
                     <property name="text">
                      <string>All</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QComboBox" name="comboBoxAge">
                    <item>
                     <property name="text">
                      <string>All</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>Home Address:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <widget class="QComboBox" name="comboBox_3">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QComboBox" name="comboBox_4">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="0">
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>Business Address:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="5" column="0">
                   <widget class="QComboBox" name="comboBox_6">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="5" column="1">
                   <widget class="QComboBox" name="comboBox_7">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="6" column="0">
                   <widget class="QCheckBox" name="checkBoxUserSearchOnlineUsers">
                    <property name="text">
                     <string>Online Users</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="7" column="0">
                   <spacer name="verticalSpacer">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>0</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item row="6" column="1">
                   <widget class="QCheckBox" name="checkBoxUserSearchWebcamUsers">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Webcam Users</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageUsersSearchResult">
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="1" column="0">
            <widget class="QTableView" name="tableViewUsersResult">
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="pushButtonPreviousUsersResultPage">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Previous</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonNextUsersResultPage">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Next</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonUserDetails">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Details</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonAddAsContact">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="toolTip">
                <string>Add as a contact</string>
               </property>
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="labelUsersResult">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Result</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabGroups">
      <attribute name="title">
       <string>Groups</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="QStackedWidget" name="stackedWidgetGroups">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="pageGroupsSearchCondition">
          <layout class="QGridLayout" name="gridLayout_14">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBoxGroupsSearchMode">
             <property name="title">
              <string>Search Mode</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="0" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <spacer name="horizontalSpacer_10">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QRadioButton" name="radioButtonGroupsMatchExactly">
                  <property name="text">
                   <string>&amp;Exactly Match</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_11">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QRadioButton" name="radioButtonGroupsMatchWildcard">
                  <property name="text">
                   <string>&amp;Wildcard Match</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_12">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string/>
             </property>
             <layout class="QGridLayout" name="gridLayout_10">
              <item row="0" column="0">
               <widget class="QStackedWidget" name="stackedWidgetGroupsSearchCondition">
                <property name="currentIndex">
                 <number>1</number>
                </property>
                <widget class="QWidget" name="pageGroupsSearchMatchWildcard">
                 <layout class="QGridLayout" name="gridLayout_11">
                  <item row="0" column="0">
                   <layout class="QGridLayout" name="gridLayout_12">
                    <item row="0" column="0">
                     <widget class="QLabel" name="labelCreator">
                      <property name="text">
                       <string>Creator:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QLineEdit" name="lineEditCreator"/>
                    </item>
                    <item row="0" column="2">
                     <widget class="QLabel" name="labelGroupType">
                      <property name="text">
                       <string>Type:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="3">
                     <widget class="QComboBox" name="comboBoxGroupType"/>
                    </item>
                   </layout>
                  </item>
                  <item row="1" column="0">
                   <spacer name="verticalSpacer_3">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>151</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="pageGroupsSearchMatchExactly">
                 <layout class="QGridLayout" name="gridLayout_13">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>Group ID:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLineEdit" name="lineEditGroupID"/>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="labelGroupName">
                    <property name="text">
                     <string>Group Name:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLineEdit" name="lineEditGroupName"/>
                  </item>
                  <item row="4" column="0">
                   <spacer name="verticalSpacer_4">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageGroupsSearchResult">
          <layout class="QGridLayout" name="gridLayout_16">
           <item row="1" column="0">
            <widget class="QTableView" name="tableViewGroupsResult"/>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QPushButton" name="pushButtonPreviousUsersResultPage_2">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Previous</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonNextUsersResultPage_2">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Next</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonGroupDetails">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Details</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonJoinGroup">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="toolTip">
                <string>Add as a contact</string>
               </property>
               <property name="text">
                <string>Join</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="labelGroupsResult">
             <property name="text">
              <string>Result</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCondition">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Condition</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSearch">
       <property name="text">
        <string>&amp;Search</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonFinish">
       <property name="text">
        <string>&amp;Finish</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonFinish</sender>
   <signal>clicked()</signal>
   <receiver>SearchClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
